common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -219,6 +219,10 @@ def modulestore():
             from lms.djangoapps.ccx.modulestore import CCXModulestoreWrapper
             _MIXED_MODULESTORE = CCXModulestoreWrapper(_MIXED_MODULESTORE)
 
+        for wrapper_path in settings.MODULESTORE_WRAPPERS:
+            wrapper = load_function(wrapper_path)
+            _MIXED_MODULESTORE = wrapper(_MIXED_MODULESTORE)
```

Comment:
hows the handling here if a wrapper can't be found or crashes on initialization?  will it block app startup?